HealthcareSystem/UserService/Model/Email.cs

```diff
@@ -26,10 +27,11 @@ public override int GetHashCode()
 
         private void Validate()
         {
-            if (!IsRegular(Value)) throw new Exception("Invalid email");
+            if (string.IsNullOrEmpty(Value)) throw new ValidationException("Email does not exist!");
+            if (!IsGoodFormat(Value)) throw new ValidationException("Invalid email");
         }
 
-        private bool IsRegular(string Value)
+        private bool IsGoodFormat(string Value)
```

Comment:
@draganacarapic named a similar method IsValidFormat or something like that on another branch, one of you should change so it matches.